src/doc/book/macros.md

```diff
@@ -509,7 +509,7 @@ A macro defined within the body of a single `fn`, or anywhere else not at
 module scope, is visible only within that item.
 
 If a module has the `macro_use` attribute, its macros are also visible in its
-parent module after the child’s `mod` item. If the parent also has `macro_use`
+parent module after its `mod` item. If the parent also has `macro_use`
```

Comment:
Now I agree, `its` introduces the ambiguity problem. In fact, I made an analogy with the first sentence and the second one

|  |   | |
| ------------- |:-------------:| -----:|
|a module(it) |its parent|the child’s |
|the parent |the grandparent |the parent's|
and I thought the second one was better, and got `a module`==`the child`, so I changed `the child's` to `its`. 

Thanks again.